.github/workflows/pull_request.yml

```diff
@@ -3,38 +3,38 @@ jobs:
   arrange:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
         with:
-          go-version: '1.16'
-      - run: go get github.com/jdeflander/goarrange
+          go-version: '1.21'
```

Comment:
[Go 1.22](https://tip.golang.org/doc/go1.22) is the latest.

Reply:
The only reason we bumped the version is because `1.21` supports `slog`, no need to check later versions, they should be backward compatible.

Reply:
I got it; it makes sense. If the logger is passed via an interface, there's no need to depend even on the `slog` package; perhaps this upgrade is unnecessary.

Another thought: if the server constructor change is accepted, the existing contract will be changed, and ideally, a new release should be created not to break people's project builds. Why not bump up the Golang version if a new release is made then?

Reply:
Yes, we should create a new release.